userguide/tutorials/aws-single-ami.adoc

```diff
@@ -199,64 +199,76 @@ In order to install it:
 
 === SSL Certificates
 
-We have configured `nginx` to listen to port `443` (and forward traffic to KAUI). By default, accessing the service from a web browser will show a `Not Secure` site. In order to make the site secure, you will need to add a valid certificate. The easiest option to add the certificate is to rely on `Let’s Encrypt`, a Certificate Authority (CA) that provides an easy way to obtain and install free TLS/SSL certificates.
+`nginx` is configured to listen on port 443 for messages to be forwarded to KAUI. Port 443 is the standard port for HTTPS. By default, accessing the service from a web browser will show a `Not Secure` site. In order to make the site secure, you will need to add a valid certificate. The easiest option to add the certificate is to make use of the tool `certbot`, which relies on the free Certificate Authority (CA) `Let’s Encrypt`  This method provides an easy way to obtain and install free TLS/SSL certificates.
 
-The steps to add the certifcate are fairly simple and rely on a tool called `certbot`.
+`Certbot` is pre-installed in the latest Kill Bill AMI. However, we now recommend using a different version of `certbot` available through the new package manager `snap`. This package manager is also preinstalled in the Kill Bill AMI.
```

Comment:
Could you explain why the version pre-installed is not adequate ? Also, we typically install packages using `apt-get`, is `snap` really required. @pierre Any thoughts?

Reply:
I am not sure if this is a better approach for Kill Bill or not. I suggested it because it is now the recommended procedure on the Certbot web site and it may be a little simpler than the previous method. I can change it to the older instructions if you prefer, but I have not tested them yet.